src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+:ruby
```

Comment:
I believe splitting this view in some partials would make it more readable. It could be one partial per card (`.card`)

Reply:
I don't really see the point of having one partial per card, which would be a few lines and a call to another partial. I have just added one partial for the last table (patchinfos).

Reply:
No worries